CopyOfapp.R

```diff
@@ -0,0 +1,134 @@
+#rm(list=ls())
```

Comment:
This file can probably be removed before merge? No need to keep the copy, as if it should ever be needed, it can be checked from Github history.